Entity usage plugin for media references in Gutenberg

chindris opened the pull request:
## Package(s) involved

silverback_gutenberg

## Description of changes

Adds en Entity Usage plugin for media embedded in Gutenberg.

## Related Issue(s)

[Create Gutenberg plugin for Entity Usage module](https://github.com/AmazeeLabs/silverback-mono/issues/827)

## How has this been tested?

Locally, manual.

chindris committed:
chore: entity usage plugin for media references in gutenberg

pmelab reviewed (changes requested):
Thanks! The implementation looks on point. But I think we should have a basic test that makes sure it still works. When for example the block parser changes and its not called `mediaEntityIds` any more.

Here are the other tests implement by @Leksat :
https://github.com/AmazeeLabs/silverback-mono/tree/development/packages/tests/silverback-gutenberg/playwright-tests

chindris committed:
chore: added the entity usage module to drupal.

chindris committed:
chore: update the install cache and default content

chindris committed:
chore: integration tests for gutenberg entity usage

chindris committed:
Merge branch 'development' into feature/entity-usage-plugin

chindris commented:
> Thanks! The implementation looks on point. But I think we should have a basic test that makes sure it still works. When for example the block parser changes and its not called `mediaEntityIds` any more.
> 
> Here are the other tests implement by @Leksat : https://github.com/AmazeeLabs/silverback-mono/tree/development/packages/tests/silverback-gutenberg/playwright-tests

Hi @pmelab !
I added a set of playwright tests and also updated the silverback-drupal app to contain the entity usage module.

chindris committed:
chore: configuration update

pmelab commented:
Awesome! Thanks @chindris !
